src/main/java/org/apache/accumulo/testing/performance/tests/ConditionalMutationsPT.java

```diff
@@ -214,9 +214,10 @@ private static double randomizeConditionalMutationsTime(ConditionalWriter cw, Lo
     if (cmuts.size() != count) {
       throw new RuntimeException();
     }
-    long t2 = System.currentTimeMillis();
 
-    return 10000 / ((t2 - t1) / 1000.0);
+    long t2 = System.nanoTime();
+
+    return 10000 / TimeUnit.MICROSECONDS.toSeconds(t2 - t1);
```

Comment:
Also should be `NANOSECONDS`

Reply:
Same reasoning for the rest of these too.